README.md

```diff
@@ -1,17 +1,120 @@
+conversation with the computer.
+
+When you see a shell it will be at a "prompt." This means, from the computer's
+perspective "I'm listening......" While prompts may look different from
+computer to computer (they're customizable and can hold the data you find
+helpful: date, time, name of directory you're in, etc.), they all have the same
+essential function of ferrying your command to the operating system and running
+it.
+
+You can ask the computer to do a great variety of tasks: some or simple file
+manipulation that you might recognize as soemthing you do with a mouse: moving
+a file, renaming a file, making a folder (or subdirectory, as shell-masters
+call it).
+
+You can ask the computer to run programs like `ruby my_awesome_program.rb` or
+even to launch an application!
+
+You can even ask the computer to do data analytics and data collation work
+like: search through all the files in a directory and print out the lines that
+contain the word "poodle" and then count the total number of vowels seen in
+all those lines.
+
+It's cases like this last one that allow you to move from having your computer
+define your workflow in a way that makes _it_ happy to defining workflows that
+your computer should do to make _you_ happy. You'll get there!
+
+## Recognize a Shell Interface
+
+A "shell" interface looks like the following pictures below.
+
+Typically you'll be given some text output and a prompt. The prompt will
+receive commands. You tell the shell you're done typing and that it should
+consider your request by hitting `return` or `enter`. After receiving your
+command the shell will respond with the information you requested, or will
+signal that it did your bidding (say renaming a file), or it will print an
+error message. Then you'll be given another prompt and can do it again, and
+again, and again.... This cycle can be called a "prompt-evaluation-response
+cycle."
+
+Here's the Learn In-Browser IDE's shell. Since it's so important for our
+Learn-based learners, it's always open. It is inside this shell that you will
+issue commands like `learn` and `learn submit`.
+
+![Learn In-Browser IDE Shell](https://curriculum-content.s3.amazonaws.com/web-development/intro-the-shell/learn_shell.png)
+
+Here's the Terminal application in MacOSX running a shell. You can launch it by
+navigating with the Finder to `Applications` &rarr; `Utilities` &rarr;
+`Terminal`.
+
+![MacOSX Shell](https://curriculum-content.s3.amazonaws.com/web-development/intro-the-shell/osx_shell.png)
+
+You may sometimes hear the shell referred to as your "command line" or
+"terminal" or "console". There are some subtle differences between each of
+those terms, but by and large you can think of these as referring to ways of
+interacting with the Shell.
+
+## Recognize Operating Systems Providing a Shell
+
+Surprisingly, most operating systems (MacOS, Windows, and Linux / Unix) provide
+a shell! You might be surprised that even Windows has preserved a shell-like
```

Comment:
I'm not sure that new users even know that Windows is a hard system to be a developer on. I consistently had students asking my why most developers use a Mac, and why Windows wasn't a good thing to use. Maybe consider re-writing this to tell users that they might not have even known that all current operating systems include a shell program?

Reply:
That's definitely what this was trying to communicate.